content/applications/general/email_communication/email_servers.rst

```diff
@@ -5,122 +5,122 @@ Send and Receive Emails in Odoo with an Email Server
+
+You can use an G Suite server for any Odoo hosting type. To do so you need to setup the SMTP relay
+service. The configuration steps are explained in `Google documentation <https://support.google.com
+/a/answer/2956491?hl=en>`__.
 
 Restriction
 -----------
-Please note that port 25 is blocked for security reasons on our SaaS and Odoo.sh platform. Try using
-465, 587, or 2525.
+
+Please note that port 25 is blocked for security reasons on our SaaS and Odoo.sh platform. Try
+using 465, 587, or 2525.
 
 .. _email_communication/default_from:
 
 Use a default "From" email address
 ----------------------------------
 
-Sometimes, an email's "From" (outgoing) address can belong to a 
-different domain, and that can be a problem.
+Sometimes, an email's "From" (outgoing) address can belong to a different domain, and that can be a
+problem.
 
-For example, if a customer with address *mary@customer.example.com* responds to
-a message, Odoo will try to redistribute that same email to other subscribers
-in the thread. But if the domain *customer.example.com* forbids that kind of
-usage for security, the Odoo's redistributed email would get
-rejected by some recipients' mail servers.
+For example, if a customer with address *mary\@customer.example.com* responds to a message, Odoo
+will try to redistribute that same email to other subscribers in the thread. But if the domain
+*customer.example.com* forbids that kind of usage for security, the Odoo's redistributed email
+would get rejected by some recipients' mail servers.
 
-To avoid those kind of problems, you should make sure all emails use a "From"
-address from your authorized domain.
+To avoid those kind of problems, you should make sure all emails use a "From" address from your
+authorized domain.
 
-If your MTA supports `SRS (Sender Rewriting Scheme)
-<https://en.wikipedia.org/wiki/Sender_Rewriting_Scheme>`_, you can enable it
-to handle these situations. However, that is more complex and requires more
-technical knowledge that is not meant to be covered by this documentation.
+If your MTA supports `SRS (Sender Rewriting Scheme) <https://en.wikipedia.org/wiki/Sender_Rewriting
+_Scheme>`_, you can enable it to handle these situations. However, that is more complex and
+requires more technical knowledge that is not meant to be covered by this documentation.
 
 Instead, you can also configure Odoo to do something similar by itself:
 
-#.  Set your domain name in the General Settings.
+#. Set your domain name in the :guilabel:`General Settings`.
 
-    .. image:: email_servers/alias-domain.png
-       :align: center
+   .. image:: email_servers/alias-domain.png
+      :align: center
+      :alt: Setting the domain alias configuration on Odoo.
 
-#.  In developer mode, go to :menuselection:`Settings --> Technical -->
-    Parameters --> System Parameters`.
+#. In developer mode, go to :menuselection:`Settings --> Technical --> Parameters --> System
+   Parameters`.
```

Comment:
- Add a line before `` #. In developer mode...Parameters`. `` that says `` #. Turn on :doc:`developer mode </applications/general/developer_mode>` ``
- Delete `In developer mode`
- Capitalize the `g` in `go to`